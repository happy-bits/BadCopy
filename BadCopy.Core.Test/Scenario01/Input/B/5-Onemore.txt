
	eee
	eee
	rrr
	#region solution
		sss
		ddd

		fff
	#endregion
	zzz
	zzz
	xxx