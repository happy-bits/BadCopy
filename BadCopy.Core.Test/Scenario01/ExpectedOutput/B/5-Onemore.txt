
	eee
	eee
	rrr
	zzz
	zzz
	xxx